src/util.ts

```diff
@@ -307,7 +307,7 @@ function sideloadManifest(application: string, manifestPath: string): Promise<an
       }
 
       const [parsedType, parsedGuid, parsedVersion] = await parseManifest(manifestPath);
-      await addManifest(application, manifestPath);
+      await addManifest(application, parsedGuid, manifestPath);
```

Comment:
manifestId rename here too...

Reply:
I think the variable name here is making sense. That means the id is from parsing manifest.